### PR TITLE
Fix/warnings

### PR DESCRIPTION
**Pull request by hakuturu583**
## Types of PR

- [ ] New Features
- [ ] Upgrade of existing features
- [x] Bugfix

## Description

- suppress warnings from boost geometry
- suppress warnings from setuptools
- fix -Wrange-loop-construct warnings

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,9 @@
 
 # external packages
 external/autoware_auto_msgs
+external/autoware_common/
+external/autoware_msgs/
+external/color_names/
 external/tier4_msgs
 external/zmqpp_vendor
```
Comment (HansRobo) [inline .gitignore:19]: I think this is simpler and less maintenance 
```suggestion
external/*
!external/lanelet2_matching
!external/concealer
```

**Review by HansRobo — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (hakuturu583) [reply .gitignore]: Fixed in https://github.com/tier4/scenario_simulator_v2/pull/951/commits/71e022ca98c3bea73375a8c5eea10d73db1a61cb

Comment (HansRobo) [reply .gitignore]: OK!

**Review by HansRobo — APPROVED** [review]
LGTM pending regression test

**Pull request merged**